Vagrantfile

```diff
@@ -9,26 +9,31 @@ Vagrant.configure(2) do |config|
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
 
-  # Create port forward mappings
-  # These are not normally required, since all access happens on the
-  # 192.168.77.77 IP address
-  #
-  # config.vm.network "forwarded_port", guest: 80, host: 80
-  # config.vm.network "forwarded_port", guest: 443, host: 443
-  # config.vm.network "forwarded_port", guest: 4443, host: 4443
-  # config.vm.network "forwarded_port", guest: 8501, host: 8501
-
   vm_memory = ENV.fetch('VM_MEMORY', 10 * 1024).to_i
   vm_cpus = ENV.fetch('VM_CPUS', 4).to_i
 
+  os=`uname`.strip
+  if os == 'Darwin'
+    default_if=`route get default | grep interface`.split(" ").last
+  else
+    default_if=`/sbin/route | grep default`.split(" ").last
```

Comment:
This point is still relevant after refactoring I did: See https://github.com/SUSE/scf/blob/d465164bea964412c34ddce4c7ef1b704bf20071/Vagrantfile#L15

My thoughts were that we're going to be shelling out either way. I get the point about sticking to ruby, but since I'll be using snippets from here for the non-vagrant CI deploys (mostly bash), I went with whatever was easiest (also the mixed-language approach is shorter and more readable to me). Let's talk more about it on Tuesday.